### PR TITLE
Add setup files:Milestone 1

### PR DESCRIPTION
**Pull request by pasytchangwa**
In this pull request, the following was done: 

- A react app was set up locally
- React documentation was used
- A development and feature branches were created as per Git flow requirements
- The react app was push to the remote repository

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello world
```
Comment (arslanbisharat) [inline README.md:1]: - [ ] Please add descriptive Readme. This is one of the basics requirements here at Microverse. Please go back and have look at the project requirements.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ] - [ ] Please add descriptive Pull Request description. This is one of the basics requirements here at Microverse. Please go back and have look at the project requirements. 

- [ ] - [ ] Please add descriptive commits. This is one of the basics requirements here at Microverse. Please go back and have look at the project requirements. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (Dipeshtwis) [inline README.md:1]: - [ ] Still, this issue is not resolved. If you are struggling, you can use [microverse template](https://github.com/microverseinc/readme-template) to make a descriptive and professional README. :slightly_smiling_face:

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @pasytchangwa, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**

-------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**